lib/src/command.dart

```diff
@@ -288,6 +290,34 @@ and include the logs in an issue on https://github.com/dart-lang/pub/issues/new
     }
     _command = list.join(' ');
   }
+
+  /// Enables the pubspec override feature for this command.
+  ///
+  /// This adds the `--pubspec-overrides` option to the command and enables
+  /// the pubspec override feature for the package for which the command
+  /// is being executed.
+  void allowPubspecOverrides() {
+    assert(!_allowPubspecOverrides);
+    _allowPubspecOverrides = true;
+
+    argParser.addOption(
+      'pubspec-overrides',
```

Comment:
Let's not have a command line option

Reply:
I'll find some time to review more later..

Reply:
Done